### PR TITLE
[Fix] console errors on top nav menu

### LINKED ISSUE #1383: Main nav bar errors
**Describe the bug**
Check the logs on admin page. There are currently alot of console logs errors on the admin page that we should look into. Here is a file for the errors:

[admin-errors.log](https://github.com/CDH-Studio/I-Talent/files/6502138/admin-errors.log)

edit: the errors seem to be coming from the navigation bar and not from the admin page
**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by mo-5**
#### ⭐ Changes introduced

There were some key related errors when you click on the menu dropdown. 

#### 🔗 Related issue(s)

closes #1383

#### ☑️ Checklist
If the database has been modified:
- [ ] Update database diagram 
- [ ] Create new migration 

If API endpoints has been modified:
- [ ] Update backend documentation 
- [ ] Create/Update validators for the API endpoints 
- [ ] The API endpoints are properly secured with keycloak 

If the UI has been modified:
- [ ] It is translated in both language (with no hard coded text) 
- [ ] The modifications are tab friendly
- [ ] It is accessible

If translations have been modified:
- [ ] run `yarn i18n:validate" and clear errors

### CODE REVIEW

File: services/frontend/src/components/layouts/appLayout/topNav/TopNavView.jsx [hunk 1]
```diff
@@ -57,50 +56,47 @@ const TopNavView = ({ isAdmin, loading, displaySearch, displayLogo, intl }) => {
     <Menu
       // Ant-design issue: recognizes either depending on machine
       // "-1" is used for off-screen content that appears on a specific event
-      /* eslint-disable react/jsx-no-duplicate-props */
       tabIndex={-1}
-      tabindex="-1"
```
Comment (dynamic11) [inline services/frontend/src/components/layouts/appLayout/topNav/TopNavView.jsx:62]: this was added by suku and benoit for accessibility reasons. are we sure we want to remove it? im ok with removing but once again @Rizvida1 should review this

**Review by dynamic11 — CHANGES_REQUESTED** [review]
please review with @Rizvida1

**Review by Rizvida1 — APPROVED** [review]

File: services/frontend/src/components/layouts/appLayout/topNav/TopNavView.jsx (hunk 1 above)
Comment (Rizvida1) [reply services/frontend/src/components/layouts/appLayout/topNav/TopNavView.jsx:62]: It works as it should

File: services/frontend/src/components/layouts/appLayout/topNav/TopNavView.jsx [hunk 2]
```diff
@@ -57,50 +56,47 @@ const TopNavView = ({ isAdmin, loading, displaySearch, displayLogo, intl }) => {
     <Menu
       // Ant-design issue: recognizes either depending on machine
       // "-1" is used for off-screen content that appears on a specific event
```
Comment (dynamic11) [inline services/frontend/src/components/layouts/appLayout/topNav/TopNavView.jsx]: these comments hould be removed

**Review by dynamic11 — APPROVED** [review]
LGTM 🚀

**Pull request merged**